keps/sig-node/2625-cpumanager-policies-thread-placement/README.md

```diff
@@ -0,0 +1,346 @@
+| Baremetal, 2-way SMT capable (x86_64), SMT enabled   | 2                          |
+| Baremetal, 2-way SMT capable, SMT disabled           | 1                          |
+
+SMT can usually be disabled via software (kernel parameter, firmware settings), and [other implementations than 2-way SMT are possible](https://en.wikipedia.org/wiki/Simultaneous_multithreading).
+Hence, the ratio between virtual and physical CPUs is not fixed and is not predictable, subject to changes in future.
+Furthermore, this new unit would make sense only for CPU resources, and not for all the other resources.
+
+### Future extension
+
+We would like to mention a further extension of this work, which we are *not* proposing here.
+
+A further subset of the latency sensitive class of workload we identified (CNF, HFT) benefits most of non-SMT system, delivering the best possible performance here.
+For these applications, just disabling SMT at machine level solves the need of the workload, but overall creates worse usage of hardware resources and poorer container density.
+
+Another policy option, or a further refinement of `reject-non-smt-aligned`, which enables non-SMT emulation on SMT-enabled system would allow to accommodate these needs, but this would cause even more significant resource accounting mismatches
+as described above. Furthermore, at the moment of writing we are still assessing how large is the set of the classes which benefit of these extra guarantees.
+
+For all these reasons we postponed this work to a later date.
+
+### Test Plan
+
+The [implementation PR](https://github.com/kubernetes/kubernetes/pull/101432) will extend both the unit test suite and the E2E test suite to cover the policy changes described in this KEP.
+
+### Graduation Criteria
+
+#### Alpha
+- [X] Implement the new policy.
+- [X] Ensure proper e2e node tests are in place.
+
+#### Alpha to Beta Graduation
+- [X] Gather feedback from the consumer of the policy.
+- [X] No major bugs reported in the previous cycle.
+
+#### Beta to G.A Graduation
+- [X] Allowing time for feedback (1 year).
+- [X] Risks have been addressed.
+
+### Upgrade / Downgrade Strategy
+
+We expect no impact. The new policies are opt-in and separated by the existing ones.
+
+### Version Skew Strategy
+
+No changes needed
+
+## Production Readiness Review Questionnaire
+### Feature enablement and rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [X] Feature gate (also fill in values in `kep.yaml`).
+    - Feature gate name: `CPUManagerPolicyOptions`.
+    - Components depending on the feature gate: kubelet
+  - [X] Change the kubelet configuration to set the CPUManager policy to `configurable`
+  - [X] Change the kubelet configuration adding the CPUManager policy option to `reject-non-smt-aligned`
+* **Does enabling the feature change any default behavior?**
+  - Yes, it makes the behaviour of the CPUManager static policy more restrictive and can lead to pod admission rejection.
```

Comment:
note for others that cpu manager static policy can already reject pods, this just adds another reason for said rejection.